profiles/base.yaml

```diff
@@ -382,6 +382,7 @@ storage_containers:
 gear:
 gear_sets:
 empath_healing:
+heal_threshhold: 25
```

Comment:
What do you think about `safe_room_health_threshold`? I'm looking for something to indicate how or why it is used.